src/scripts/rank_SVs.pl

```diff
@@ -61,7 +61,7 @@
 	if($fg1 gt $fg2){
 		my $tmp = $fg1;
```

Comment:
This would make the swap of variables clearer and more compact.
```suggestion
		($fg1, $fg2) = ($fg2, $fg1)
```